python/lsst/sdm_schemas/schemas/dp02_dc2.yaml

```diff
@@ -103,4890 +135,6840 @@ tables:
+    tap:column_index: 2153
   - name: g_calib_photometry_used
     "@id": "#Object.g_calib_photometry_used"
     datatype: boolean
     description: Propagated from sources. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2154
   - name: g_calib_psf_candidate
     "@id": "#Object.g_calib_psf_candidate"
     datatype: boolean
     description: Propagated from sources. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2155
   - name: g_calib_psf_reserved
     "@id": "#Object.g_calib_psf_reserved"
     datatype: boolean
     description: Propagated from sources. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2156
   - name: g_calib_psf_used
     "@id": "#Object.g_calib_psf_used"
     datatype: boolean
     description: Propagated from sources. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2157
   - name: g_centroid_flag
     "@id": "#Object.g_centroid_flag"
     datatype: boolean
     description: General Failure Flag. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2158
   - name: g_centroid_x
     "@id": "#Object.g_centroid_x"
     datatype: double
     description: Centroid from SDSS Centroid algorithm. Measured on g-band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 2159
   - name: g_centroid_xErr
     "@id": "#Object.g_centroid_xErr"
     datatype: float
     description: 1-sigma uncertainty on x position. Measured on g-band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 2160
   - name: g_centroid_y
     "@id": "#Object.g_centroid_y"
     datatype: double
     description: Centroid from SDSS Centroid algorithm. Measured on g-band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 2161
   - name: g_centroid_yErr
     "@id": "#Object.g_centroid_yErr"
     datatype: float
     description: 1-sigma uncertainty on y position. Measured on g-band.
     fits:tunit: pixel
+    tap:principal: 0
+    tap:column_index: 2162
   - name: g_decl
     "@id": "#Object.g_decl"
     datatype: double
     description: Position in Declination. Measured on g-band.
     fits:tunit: deg
     ivoa:ucd: pos.eq.dec
+    tap:principal: 0
+    tap:column_index: 2002
   - name: g_extendedness
     "@id": "#Object.g_extendedness"
     datatype: double
     description: Set to 1 for extended sources, 0 for point sources. Measured on g-band.
     fits:tunit:
+    tap:principal: 1
+    tap:column_index: 2163
   - name: g_extendedness_flag
     "@id": "#Object.g_extendedness_flag"
     datatype: boolean
     description: Set to 1 for any fatal failure. Measured on g-band.
     fits:tunit:
+    tap:principal: 0
+    tap:column_index: 2164
   - name: g_free_cModelFlux
     "@id": "#Object.g_free_cModelFlux"
     datatype: double
     description: Flux from the final cmodel fit. Measured on g-band.
     fits:tunit: nJy
+    tap:principal: 0
```

Comment:
We need to note somewhere that "free" here means that the models was fit full in the (in this case) `g` band, and the columns without "free" had only their flux fit in `g` and the rest held fixed from the fit in the reference band.

Reply:
What do you think about swapping "free" and "cModel" in future versions of this?

Reply:
:+1: to swapping "free" and "cModel".